src/aria/widgets/controllers/reports/ControllerReport.js

```diff
@@ -36,6 +36,12 @@ module.exports = Aria.classDefinition({
         this.cancelKeyStroke = false;
 
         /**
+         * Controller specifies if the keystroke's default behavior has to be canceled
+         * @type Boolean
+         */
+        this.cancelKeyStrokeDefaultBahevior = false;
```

Comment:
It would be better to use the correct spelling for `Behavior` in the name of the property (this should be changed in all places where the variable is used).